mypyc/irbuild/ll_builder.py

```diff
@@ -755,6 +756,27 @@ def make_dict(self, key_value_pairs: Sequence[DictEntry], line: int) -> Value:
 
         return result
 
+    def new_list_op(self, values: List[Value], line: int) -> Value:
+        length = self.add(LoadInt(len(values), line, rtype=c_int_rprimitive))
+        empty_list = self.call_c(new_empty_list_op, [length], line)
+        if len(values) == 0:
+            return empty_list
+        args = [self.coerce(item, object_rprimitive, line) for item in values]
+        ob_item_ptr = self.add(GetElementPtr(empty_list, PyListObject, 'ob_item', line))
+        ob_item_base = self.add(LoadMem(pointer_rprimitive, ob_item_ptr, empty_list, line))
+        for i in range(len(values)):
+            if i == 0:
+                item_address = ob_item_base
+            else:
+                offset = self.add(LoadInt(PLATFORM_SIZE * i, line, rtype=pointer_rprimitive))
+                item_address = self.add(BinaryIntOp(pointer_rprimitive, ob_item_base, offset,
+                                                    BinaryIntOp.ADD, line))
+            self.add(SetMem(object_rprimitive, item_address, args[i], empty_list, line))
+        return empty_list
+
+    def new_set_op(self, values: List[Value], line: int) -> Value:
+        return self.call_c(new_set_op, [], line)
```

Comment:
This isn't passing the values on?

Reply:
that's simply because `new_set_op` takes no argument, so `values` here is just an empty list.